python_modules/dagster/dagster/_config/config_type.py

```diff
@@ -347,7 +347,7 @@ def select_color(context):
         """
         if name is None:
             name = enum.__name__
-        return cls(name, [EnumValue(v.name, python_value=v) for v in enum])
+        return cls(name, [EnumValue(v.name, python_value=v.value) for v in enum])
```

Comment:
Let's make a new method instead of changing this one. There very well might be someone somewhere calling this. No reason to risk it. Just add a new method.